staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -1367,6 +1482,16 @@ message MemoryUsage {
     int64 timestamp = 1;
     // The amount of working set memory in bytes.
     UInt64Value working_set_bytes = 2;
+    // Available memory for use.  This is defined as the memory limit = workingSetBytes.
```

Comment:
s/defined as the memory limit = workingSetBytes/defined as the memory limit - workingSetBytes.

xref: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L215

Reply:
good catch!